### ISSUE TITLE
[Feature Request] - Support `noColor` in launcher.

### ISSUE THREAD

**Issue by ckipp01** [scalacenter/bloop#1478]
Depending on the client, they may not support color in the way they are displaying test output during a debug session. For example, using [Vimspector](https://github.com/puremourning/vimspector) via coc-metals, the output is showing the escape codes. There are ways to disable color output via the cli etc, but as far as I can tell, there isn't a way to disable the colored logging, but also it seems that the `baseBspLogger` is actually created with it turned off anyways?

https://github.com/scalacenter/bloop/blob/f9fd96b8fa655474485cfc91cead432e7cfb69bf/frontend/src/main/scala/bloop/bsp/BloopBspServices.scala#L98

I don't fully see where it's then getting turned on, or what would be the recommended way to turn color off for DAP communication.

Basically, I'm trying to avoid this:
<img width="463" alt="Screenshot 2020-09-12 at 10 26 55" src="https://user-images.githubusercontent.com/13974112/92994948-39b19a80-f4ff-11ea-9bc6-649ddee544a6.png">

Any thoughts?

_Originally posted by @ckipp01 in https://github.com/scalacenter/bloop/discussions/1372_

**Comment by tgodzik** [comment]
Btw. you should also be able to use `NO_COLOR` env variable to work around it.

**Comment by ckipp01** [comment]
> Btw. you should also be able to use `NO_COLOR` env variable to work around it.

Yup, that works, but also turns color off when using Bloop cli, which I don't want. Actually literally as typing this I'm not thinking this all may be problematic in general. If we have this in launcher and we say `noColor`, and then launch bloop with that. If you then use Bloop cli, that reporter with `noColor` set will still be used won't it?

**Comment by tgodzik** [comment]
@ckipp01 uch, I think it will. We would need a per client setting.

**Comment by ckipp01** [comment]
> @ckipp01 uch, I think it will. We would need a per client setting.

😆 your idea to do this in Metals is looking more and more appealing to me.

**Comment by tgodzik** [comment]
> > @ckipp01 uch, I think it will. We would need a per client setting.
> 
>  your idea to do this in Metals is looking more and more appealing to me.

Yeah, it seems much less complex :sweat_smile:

**Comment by tgodzik** [comment]
This one is solved in Metals, so I don't think we will be working on it in Bloop as that will require a lot more effort. We can reopen once this becomes an issue again.
